fiaas_skipper/web/api.py

```diff
@@ -32,18 +32,28 @@ def _encode(obj):
     return obj
 
 
+def _force_bootstrap(request):
+    if request.get_json() and 'force_bootstrap' in request.get_json():
+        return request.get_json(force=True, silent=True)['force_bootstrap']
+
+
+def _namespaces(request):
+    if request.get_json() and 'namespaces' in request.get_json():
+        return request.get_json()['namespaces']
+
+
 @api.route('/api/deploy', methods=['POST'])
 @deploy_histogram.time()
 def deploy():
-    if request.get_json() and 'namespaces' in request.get_json():
-        threading.Thread(target=_deploy, kwargs={'namespaces': request.get_json()['namespaces']}).start()
-    else:
-        threading.Thread(target=_deploy).start()
+    namespaces = _namespaces(request)
+    force_bootstrap = _force_bootstrap(request) or False
```

Comment:
It would be more natural for the function to return `True` or `False` directly, instead of the `or False` expression.

Reply:
Ok done.